src/vsr/replica.zig

```diff
@@ -710,6 +698,8 @@ pub fn ReplicaType(
                 .replica = replica,
             });
 
+            self.test_context = options.test_context;
```

Comment:
init calls multiversion.releases_bundled and that, in vopr, uses test_context to find `Cluster`. Anyway, assigning this early seems more direct anyway.